plugins/org.yakindu.sct.doc.user/src/user-guide/generating_code.textile

```diff
@@ -1539,10 +1589,88 @@ It is the timer service's responsibility to actually raise a time event on a sta
 
 You can conclude that in order to process time events without too much latency, the runtime environment has to call the state machine's @runCycle()@ method as frequently as needed. Consider for example a time event which is raised by the timer service after 500 ms. However, if the runtime environment calls the state machine's @runCycle()@ method with a frequency of once per 1000 ms only, the event will quite likely not be processed at the correct points in time.
 
+
 h3(#java-runtime-service). Runtime service
 
 The @RuntimeService@ class maintains all state machines that are expected to execute run-to-completion steps periodically. A client application can retrieve the @RuntimeService@ singleton using @RuntimeService.getInstance()@. It can then pause, resume or cancel all state machines that are poised to run at a specified intervall.
-Note: This is only available with the _RuntimeService_ parameter of *GeneralFeatures* set to @true@, see "Java: General Features":#java-GeneralFeatures. 
+
+Please note: To enable this feature, set the _RuntimeService_ parameter of *GeneralFeatures* to @true@, see "Java: General Features":#java-GeneralFeatures. 
+
+
+
+h2(#creating-a-new-custom-code-generator-project). Creating a new custom code generator project
```

Comment:
Is it _really_ useful to move this here? I'd link it but never move "Creating a new custom code generator project" out of the chapter "Custom code generators". That is confusing. Also, it's now a section in the chapter "Java Code generator"...?